.github/workflows/karma.yml

```diff
@@ -180,6 +180,7 @@ jobs:
             - run: yarn hydration:sauce:ci
             - run: ENABLE_SYNTHETIC_SHADOW_IN_HYDRATION=1 yarn hydration:sauce:ci
             - run: NODE_ENV_FOR_TEST=production yarn hydration:sauce:ci
+            - run: DISABLE_NATIVE_CUSTOM_ELEMENT_LIFECYCLE=1 yarn hydration:sauce:ci
```

Comment:
I confirmed manually that this was working correctly in the hydration tests. Synthetic lifecycle is used.